analyses/malawi/flooding_trigger/04_glofas_skill.md

```diff
@@ -0,0 +1,183 @@
+### Exploring GloFAS forecast skill
+
+This notebook evaluates GloFAS forecast skill at stations in Chikwawa and Nsanje. We are looking specifically at skill in predicting extreme streamflow values across various leadtimes.
+
+```python
+from datetime import datetime
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from scipy.stats import norm, pearsonr
+import numpy as np
+import pandas as pd
+import xarray as xr
+import xskillscore as xs
+from scipy.interpolate import interp1d
+from scipy.stats import rankdata
+import os
+from pathlib import Path
+import sys
+from datetime import timedelta
+from matplotlib.ticker import MaxNLocator, ScalarFormatter
+
+path_mod = f"{Path(os.path.dirname(os.path.abspath(''))).parents[1]}/"
+sys.path.append(path_mod)
+
+from src.indicators.flooding.config import Config
+from src.indicators.flooding.glofas import utils
+
+config = Config()
+mpl.rcParams['figure.dpi'] = 300
+
+PLOT_DIR = config.DATA_DIR / 'processed' / 'mwi' / 'plots' / 'flooding'
+PRIVATE_DIR = config.DATA_PRIVATE_DIR
+EXPLORE_DIR = PRIVATE_DIR / 'exploration' / 'mwi' / 'flooding'
+
+SAVE_FIG = True
+LEADTIMES = [5, 10, 15, 20, 25, 30]
+
+stations_adm2 = {
+    'G1724': 'Nsanje',
+    'G2001': 'Chikwawa'
+}
+COUNTRY_ISO3 = 'mwi'
+```
+
+Read in the processed GloFAS data and calculate the return periods. We can also plot the river discharge thresholds for each return period level.
+
+```python
+ds_glofas_reanalysis = utils.get_glofas_reanalysis(
+    country_iso3=COUNTRY_ISO3)
+ds_glofas_reforecast = utils.get_glofas_reforecast(
```

Comment:
When I run this I get an error that `Data/public/processed/mwi/glofas/mwi_cems-glofas-reforecast_v3_lt05d.nc' is missing

Reply:
Have merged in your glofas updates so should be working now.